src/OpenDB/test/gcd_abstract_lef.lefok

```diff
@@ -2,834 +2,834 @@ VERSION 5.8 ;
 BUSBITCHARS "[]" ;
 DIVIDERCHAR "/" ;
 UNITS
-    DATABASE MICRONS 1000 ;
+    DATABASE MICRONS 2000 ;
 END UNITS
 MACRO gcd
   FOREIGN gcd 0 0 ;
   CLASS BLOCK ;
   ORIGIN 0 0 ;
-  SIZE 200.26 201.6 ;
+  SIZE 100.13 100.8 ;
```

Comment:
This seems highly suspect to me, because the `SIZE` attribute is supposed to be in µm. It shouldn't have changed based on the DBU... @QuantamHD can you confirm that the size is being calculated correctly?

Reply:
After reading in the def in Klayout it reads as ~100 microns. If you look at the DEF the DB units are roughly 200,000, and the DBUnits per micron are 2000 so it should be about a 100 microns from my understanding.

Reply:
Yes, the LEF looks correct to me. My main concern is if the input DBU is for example 1000, is the output still correct?

One problem for sure is that there is a syntax error - the line needs to `SIZE [X] BY [Y]`

Reply:
If you check on your end that a different input DBU works as intended and fix the syntax error, I will approve. I think either of the sky130 platforms would be work because they use a DBU of 1000.

Reply:
The lef scaling value is set based on _dist_factor = 1.0L / db_block->getDbUnitsPerMicron();. I think this should be correct as it's set by the def file you read in. From my understanding in the LEF file the units are always in microns where as in the DEF file they are always in DBUs. If the UNITS change in the DEF it physically changes the size of the macro which is what my code models.

Reply:
Resolved Syntax issue